python/tskit/drawing.py

```diff
@@ -202,17 +207,25 @@ def draw(self):
         code as text.
         """
         dwg = svgwrite.Drawing(size=(self._width, self._height), debug=True)
-        lines = dwg.add(dwg.g(id='lines', stroke='black'))
+        default_edge_colour = "black"
+        default_node_colour = "black"
+        default_mutation_colour = "red"
+        lines = dwg.add(dwg.g(id='lines', stroke=default_edge_colour))
+        nodes = dwg.add(dwg.g(id='nodes', fill=default_node_colour))
+        mutations = dwg.add(dwg.g(id='mutations', fill=default_mutation_colour))
         left_labels = dwg.add(dwg.g(font_size=14, text_anchor="start"))
         right_labels = dwg.add(dwg.g(font_size=14, text_anchor="end"))
         mid_labels = dwg.add(dwg.g(font_size=14, text_anchor="middle"))
         for u in self._tree.nodes():
             v = self._tree.get_parent(u)
             x = self._x_coords[u], self._y_coords[u]
-            colour = "black"
-            if self._node_colours.get(u, None) is not None:
-                colour = self._node_colours[u]
-            dwg.add(dwg.circle(center=x, r=3, fill=colour))
+            fill = self._node_colours.get(u, default_node_colour)
+            if fill is not None:
+                if fill == default_node_colour:
```

Comment:
Might help to have a quick one-line comment here to explain why we're doing this. (I assume this is to minimise the size of the SVG, right?) A more compact way of expressing might be:
```
params = {} if fill == default_node_colour else {'fill': fill}

Reply:
Very neat way of doing this BTW, I like it a lot.

Reply:
Yes, that's right. Makes the SVG much cleaner. Happy to go with the one-line if-else. I assumed you didn't like these sort of python-esque contractions.